### PR TITLE
FE: basic Sign In functionality

### PR DESCRIPTION
**Pull request by bohdanvan**
basic-sign-in

### CODE REVIEW

**Comment by bohdanvan** [comment]
@bohdanvan @js_mentor чекни PR пліз

File: frontend/src/main/sign-out/SignOutButton.js [hunk 1]
```diff
@@ -0,0 +1,20 @@
+import { Button } from '@material-ui/core'
+import { useAuth } from 'context/Auth'
+import React from 'react'
+import authApi from 'api/AuthApi'
+
+const SignOutButton = (props) => {
+    const [auth, setAuth] = useAuth()
+
+    const handleClick = async () => {
+        // @ts-ignore
```
Comment (dmitrychernykh) [inline frontend/src/main/sign-out/SignOutButton.js:10]: usually this is not needed

File: frontend/src/main/sign-in/SingInPage.js [hunk 2]
```diff
@@ -0,0 +1,49 @@
+import { makeStyles, Paper, Typography } from '@material-ui/core'
+import MainPaper from 'main/MainPaper'
+import React from 'react'
+import { Link } from 'react-router-dom'
+import SignInFormContainer from './SignInFormContainer'
+
+const useStyles = makeStyles((theme) => ({
+    paper: {
```
Comment (dmitrychernykh) [inline frontend/src/main/sign-in/SingInPage.js:8]: please consider using the following order for styles props if they are present:

1. position
2. position related (top, left ...) 
3. display
4. display related (flex dir, justify, align ...)
5. size related (width, height, margins, borders)
6. paddings
7. backgrounds, shadows, other graphic related
7. inner text styles (color, align, font-weight ...)
8. animations, transfers (usually animation is provided with different class from other styles)

File: frontend/src/main/MainTitle.js [hunk 3]
```diff
@@ -0,0 +1,16 @@
+import { Box, Typography } from '@material-ui/core'
+import React from 'react'
+
+const MainTitle = () => (
+    <Box display="flex"
+        height="400px"
+        alignItems="center"
+        justifyContent="center">
+        <Typography variant='h1' style={{
```
Comment (dmitrychernykh) [inline frontend/src/main/MainTitle.js]: please consider using some related class, instead of inlined styles

File: frontend/src/main/MainPaper.js [hunk 4]
```diff
@@ -0,0 +1,24 @@
+import { Paper } from '@material-ui/core'
+import React from 'react'
+import Background from './img/background.jpg'
+
+const MainPaper = ({ children }) => (
+    <Paper style={{
```
Comment (dmitrychernykh) [inline frontend/src/main/MainPaper.js]: please consider using some related class, instead of inlined styles

**Review by dmitrychernykh — APPROVED** [review]

File: frontend/src/main/sign-out/SignOutButton.js (hunk 1 above)
Comment (bohdanvan) [reply frontend/src/main/sign-out/SignOutButton.js:10]: This is a way to suppress a warning from the linter
image

**Pull request merged**